includes/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper.php

```
@@ -16,26 +16,441 @@
+		$currency             = $session->get_currency();
+		$supported_currencies = array_keys( get_woocommerce_currencies() );
+		if ( ! in_array( $currency, $supported_currencies, true ) ) {
+			throw new Exception(
+				sprintf(
+					'Checkout session %s has unsupported currency: %s.',
+					$session->get_id(),
+					$currency
+				)
+			);
+		}
+	}
```

PHPMD (2.15.0)

[error] The method validate_checkout_session is not named in camelCase. (undefined)

(CamelCaseMethodName)